pkgs/development/python-modules/nox/default.nix

```diff
@@ -29,6 +30,11 @@ buildPythonPackage rec {
     hash = "sha256-GdNz34A8IKwPG/270sY5t3SoggGCZMWfDq/Wyhk0ez8=";
   };
 
+  patches = [
```

Comment:
using fetchpatch is preferred

Reply:
The patch does not clearly apply, see the pr description.

Reply:
i see, thank for clarification